libs/sdk-core/src/lib/db/db.models.ts

```diff
@@ -1,6 +1,8 @@
 export type AccountTypes =
   | 'stash-basic'
   | 'stash-team'
+  | 'stash-developer'
+  | 'stash-startup'
```

Comment:
you'll need to add the translation for those, as we display the current plan on account home page.